Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -614,6 +614,12 @@ inline UGPRPair doWasmCall(Wasm::Instance* instance, unsigned functionIndex)
     WASM_CALL_RETURN(instance, codePtr.taggedPtr(), WasmEntryPtrTag);
 }
 
+extern "C" UGPRPair doWasmIPIntCall(Wasm::Instance* instance, unsigned functionIndex);
```

Comment:
Let's declare it in WasmSlowPaths.h. And ` REFERENCED_FROM_ASM WTF_INTERNAL` annotations are required to avoid removing this function from LTO.